mono/metadata/assembly.h

```diff
@@ -13,9 +13,11 @@ MONO_API MonoAssembly *mono_assembly_open_full (const char *filename,
 				       	MonoImageOpenStatus *status,
 					mono_bool refonly);
 MONO_API MonoAssembly* mono_assembly_load       (MonoAssemblyName *aname, 
+                                       	MonoAssembly *requesting,
                                        	const char       *basedir, 
 				     	MonoImageOpenStatus *status);
 MONO_API MonoAssembly* mono_assembly_load_full (MonoAssemblyName *aname, 
+                                       	MonoAssembly *requesting,
                                        	const char       *basedir, 
 				     	MonoImageOpenStatus *status,
```

Comment:
Both of those changes break the embedding API.

This requires new API entry points to be added to provide the missing information.